exporter/datadogexporter/translate_traces_test.go

```diff
@@ -1058,3 +1144,16 @@ func TestStatsAggregations(t *testing.T) {
 
 	assert.Equal(t, "test-version", statsVersionTag.Value)
 }
+
+// ensure that truncation helperr function truncates strings as expected
+// and accounts for the limit and multi byte ending characters
+// from https://github.com/DataDog/datadog-agent/blob/140a4ee164261ef2245340c50371ba989fbeb038/pkg/trace/traceutil/truncate_test.go#L15
+func TestTruncateUTF8Strings(t *testing.T) {
```

Comment:
I think codecov doesn't see this test because it's not on the `utils` package (where it arguably should be). Can you move it there?

Reply:
updated (along with other pr), I think codecov found it?